python/raydp/torch/torch_metrics.py

```diff
@@ -0,0 +1,44 @@
+import torchmetrics
+import sys
+module = sys.modules[__name__]
+
+class Torch_Metric():
```

Comment:
Change to `TorchMetric` will be more consistent with the style in the project.

Reply:
done.